src/lamp_py/ingestion/utils.py

```diff
@@ -215,7 +215,7 @@ def explode_table_column(table: pyarrow.table, column: str) -> pyarrow.table:
                 other_columns
             ),
         ],
-        promote=True,
+        promote_options="default",
```

Comment:
just curious what this gives you that just turning type promotion on doesnt.

Reply:
Not sure what you mean by "just turning type promotion on", could you expand on that?

Reply:
they keyword arguments for `concat_tables` changed in pyarrow v14, so `promote=True` and `promote_options="default"` are equivalent.